store/storetest/reaction_store.go

```diff
@@ -34,81 +36,72 @@ func testReactionSave(t *testing.T, ss store.Store) {
 		EmojiName: model.NewId(),
 	}
 	reaction, err := ss.Reaction().Save(reaction1)
-	if err != nil {
-		t.Fatal(err)
-	} else if saved := reaction; saved.UserId != reaction1.UserId ||
-		saved.PostId != reaction1.PostId || saved.EmojiName != reaction1.EmojiName {
-		t.Fatal("should've saved reaction and returned it")
-	}
+	require.Nil(t, err)
+
+	saved := reaction
+	assert.Equal(t, saved.UserId, reaction1.UserId, "should've saved reaction user_id and returned it")
+	assert.Equal(t, saved.PostId, reaction1.PostId, "should've saved reaction post_id and returned it")
+	assert.Equal(t, saved.EmojiName, reaction1.EmojiName, "should've saved reaction emoji_name and returned it")
 
 	var secondUpdateAt int64
 	postList, err := ss.Post().Get(reaction1.PostId, false)
-	if err != nil {
-		t.Fatal(err)
-	}
-	if !postList.Posts[post.Id].HasReactions {
-		t.Fatal("should've set HasReactions = true on post")
-	} else if postList.Posts[post.Id].UpdateAt == firstUpdateAt {
-		t.Fatal("should've marked post as updated when HasReactions changed")
-	} else {
+	require.Nil(t, err)
+
+	assert.Equal(t, postList.Posts[post.Id].HasReactions, true, "should've set HasReactions = true on post")
+	assert.NotEqual(t, postList.Posts[post.Id].UpdateAt, firstUpdateAt, "should've marked post as updated when HasReactions changed")
+
+	if postList.Posts[post.Id].HasReactions && postList.Posts[post.Id].UpdateAt != firstUpdateAt {
 		secondUpdateAt = postList.Posts[post.Id].UpdateAt
 	}
 
-	if _, err = ss.Reaction().Save(reaction1); err != nil {
-		t.Log(err)
-		t.Fatal("should've allowed saving a duplicate reaction")
-	}
+	_, err = ss.Reaction().Save(reaction1)
+	t.Log(err)
```

Comment:
Probably we no longer need the `t.Log` here, because the error message is going to show the error value anyway.